From 339a719a4049cd1b9368af1946647227ddf15390 Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Thu, 6 Feb 2020 14:38:04 -0500
Subject: [PATCH] Issue 50886 - Typo in the replication debug message

Description:  Fix typo in replication logging message

relates: https://pagure.io/389-ds-base/issue/50886

Reviewed by: mreynolds (one line commit rule)
---
 ldap/servers/plugins/replication/repl5_plugins.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ldap/servers/plugins/replication/repl5_plugins.c b/ldap/servers/plugins/replication/repl5_plugins.c
index e6b2fdb6e..caa99d1c6 100644
--- a/ldap/servers/plugins/replication/repl5_plugins.c
+++ b/ldap/servers/plugins/replication/repl5_plugins.c
@@ -622,7 +622,7 @@ multimaster_mmr_postop (Slapi_PBlock *pb, int flags)
 		break;
 	}
 	slapi_log_err(SLAPI_LOG_REPL, REPLICATION_SUBSYSTEM,
-                     "multimaster_mmr_postop - error %d for oparation %d.\n", rc, flags);
+                     "multimaster_mmr_postop - error %d for operation %d.\n", rc, flags);
 	return rc;
 }
 
-- 
2.21.1

